src/components/layout/NavBar/NavBar.styled.ts

```diff
@@ -0,0 +1,60 @@
+import { NavLink } from 'react-router';
+import styled from 'styled-components';
+
+export const Wrapper = styled.nav`
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  align-items: center;
+  padding-top: 24px;
+  width: 72px;
+  height: 100%;
+  background-color: ${({ theme }) => theme.colors.Gray_200};
+  z-index: 999;
+`;
+
+export const NavMenuList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+  align-items: center;
+`;
+
+export const NavMenu = styled.li`
+  width: fit-content;
+`;
+
+const NavMenuContentStyle = `
```

Comment:
```suggestion
import styled, { css } from 'styled-components';

const NavMenuContentStyle = css`
```
styled-components의 css 태그 함수를 사용하면 IDE의 CSS 구문 하이라이팅이나 동적 스타일링(props 기반 스타일링)이 가능해서 css 함수를 사용하는 것도 좋을 것 같습니다☺️

Reply:
지원님 피드백을 보고 `styled-components` [공식 문서](https://styled-components.com/docs/api#css)를 확인해봤는데
>A helper function to generate CSS from a template literal with interpolations. You need to use this if you return a template literal with functions inside an interpolation due to how tagged template literals work in JavaScript. If you're interpolating a string you do not need to use this, only if you're interpolating a function.

`css helper` 는 template literal 형태를 반환하는 경우에 사용하라고 하네욥!

지금 당장은 `NavMenuContentStyle` 가 문자열로만 이루어져있지만
확장성, IDE CSS 구문 강조나 오류 검사를 위해 `css` 태그를 사용하는게 좋을 거 같아요〰️

Reply:
말씀해주신대로 CSS 구문 하이라이팅 기능이 추가되는게 좋을 것 같아 css function 을 추가했습니다.